vllm/v1/worker/gpu_model_runner.py

```diff
@@ -327,6 +329,11 @@ def _may_reorder_batch(self, scheduler_output: "SchedulerOutput") -> None:
         Args:
             scheduler_output: The scheduler output.
         """
+        
+        # nothing to be reordered when the mdoel is attention free
+        if self.model_config.is_attention_free:
```

Comment:
I'm OK with this change temporarily. I'll refactor this function soon to handle both attention-free case and many other unsupported cases.